draft-bradleylundberg-cfrg-arkg.md

```diff
@@ -512,24 +513,20 @@ Then the `BL` parameter of ARKG may be instantiated as follows:
 - Elliptic curve scalar values are encoded to and from octet strings
   using the procedures defined in sections 2.3.7 and 2.3.8 of [SEC1].
 
-- `N` is the order of `crv`.
 - `G` is the generator of `crv`.
+- `N` is the order of `G`.
 
 ~~~pseudocode
 BL-Generate-Keypair() -> (pk, sk)
 
     sk = Random(1, N)
-    pk_tmp = G^sk
-    If pk_tmp equals the point at infinity, abort with an error.
-    pk = pk_tmp
+    pk = sk * G
 
 
 BL-Blind-Public-Key(pk, tau) -> pk_tau
 
     If tau = 0 or tau >= N, abort with an error.
```

Comment:
I would want to avoid aborts that are not necessary. 

For tau, you can generate it using hash_to_field() from https://www.rfc-editor.org/rfc/rfc9380.html

Alternatively, which is likely more suitable, use a try and increment approach when generating tau.